mrbgems/mruby-compiler/core/codegen.c

```diff
@@ -2202,7 +2202,7 @@ gen_retval(codegen_scope *s, node *tree)
   if (nint(tree->car) == NODE_SPLAT) {
     codegen(s, tree, VAL);
     pop();
-    genop_1(s, OP_ARYDUP, cursp());
+    genop_1(s, OP_ARYSPLAT, cursp());
   }
```

Comment:
Since this is an only usage of `OP_ARYDUP`, we should rename (if needed) instead of adding a new instruction.